app/cdap/components/GridTable/components/NestedMenu/index.tsx

```diff
@@ -72,29 +74,37 @@ export default function({
         }}
         className={classes.root}
       >
-        {menuOptions?.map((eachOption, index) => (
-          <MenuItemComponent
-            item={eachOption}
-            columnType={columnType.toLowerCase()}
-            index={index}
-            onMenuClick={handleMenuClick}
-          />
-        ))}
-        <MenuComponent
-          anchorEl={anchorEl2}
-          columnType={columnType.toLowerCase()}
-          menuOptions={nestedOptions}
-          setAnchorEl={setAnchorEl2}
-          submitOption={(e, item) => {
-            e.preventDefault();
-            e.stopPropagation();
-            setAnchorEl(null);
-            handleMenuOpenClose(title);
-            setAnchorEl2(null);
-            submitMenuOption(item.value, item.supported_dataType);
-          }}
-        />
+        {menuOptions?.map((eachOption, index) => {
+          return (
+            <MenuItemComponent
+              item={eachOption}
+              columnType={columnType.toLowerCase()}
```

Comment:
what is columntype used for?
please provide insights to me, and see if we can rename it more apprpriately

Reply:
when we will select column from header click and then try to click on menu options so only those menu options will be enabled that matches column data type